data/mods/MindOverMatter/effects/effects_psionic.json

```diff
@@ -560,7 +560,8 @@
           }
         ]
       }
-    ]
+    ],
+    "flags": [ "CRAFT_IN_DARKNESS", "READ_IN_DARKNESS" ]
```

Comment:
There’s a NOBLE HORIZON experiment log snippet where XEDRA tested what the exact capabilities of a clairsentient in complete darkness and it did not include reading. 

(The way Night Eyes works is a result of CDDA not distinguishing between very low light and complete darkness. Since no where else in the game did, I figured that having this one power do it would be janky and counterintuitive)

I have an idea for how crafting in darkness might work, but I wouldn’t assign it to Night Eyes and I need to test and see if it’s even mechanically possible.